src/calibrationlib/calibration_table_wrapper.cpp

```diff
@@ -0,0 +1,22 @@
+#include "calibration_table_wrapper.hpp"
+#include <storage/base_encoded_segment.hpp>
+
+namespace opossum {
+CalibrationTableWrapper::CalibrationTableWrapper(
+    const std::shared_ptr<Table> table, const std::string& table_name,
+    const std::vector<ColumnDataDistribution> column_data_distribution_collection)
+    : _table(table), _name(table_name), _column_data_distribution_collection(column_data_distribution_collection) {
+  assert(table->column_count() == column_data_distribution_collection.size());
```

Comment:
As that "our" assert?

Reply:
I think we use "Assert" which takes an error message.

Reply:
That's another assert. The LQP generation needs a ColumnDataDistribution per Column to generate useful predicates.